frontend/src/components/eam/eam.tsx

```diff
@@ -1,4 +1,5 @@
 import { FC } from 'react';
+import { WorkersTable } from './components/workers-table/workers-table';
```

Comment:
```suggestion
import { WorkersTable } from './components/components';
```
we already have an import